app/src/test/java/org/simple/clinic/recentpatient/AllRecentPatientsEffectHandlerTest.kt

```diff
@@ -0,0 +1,52 @@
+package org.simple.clinic.recentpatient
+
+import androidx.paging.PagingData
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.verifyNoMoreInteractions
+import org.junit.After
+import org.junit.Test
+import org.simple.clinic.TestData
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.patient.PatientRepository
+import org.simple.clinic.util.scheduler.TestSchedulersProvider
+import java.util.UUID
+
+class AllRecentPatientsEffectHandlerTest {
+
+  private val patientRepository = mock<PatientRepository>()
+  private val currentFacility = { TestData.facility() }
```

Comment:
Can we pin the facility ID we are using here?

Reply:
Unrelated, what do you think about changing the default values in `TestData` factory methods to not be random anymore? This would mean breaking some of the existing old tests, but new tests would automatically get deterministic data and consumers would need to opt into the random data usage explicitly when writing the tests.

Reply:
Yeah, that would be good. I think missed adding it here, usually, I add at least the IDs to avoid any issues. Will do the `TestData` changes in a different PR.